### PR TITLE
Added Members weights handling

### PR DESCRIPTION
**Pull request by JocelynDelalande**
Implementing the feature discussed in #94 
UI/forms is quite basic (no ajax), but that seems consistent with the app.

It introduces schema modification, but there is not yet any mechanism to migrate the schema (see #83 ) so how do we do ?

### CODE REVIEW

**Comment by almet** [comment]
I'm sorry, I've just seen this pull request right now, as I was doing some cleaning. I guess that #83 is a blocker of this very pull request, and nobody volonteered to implement it so far.

**Comment by JocelynDelalande** [comment]
@almet Hurray ! That's now 

- using migrations provided by #141
- providing up-to-date translation files

File: budget/forms.py [hunk 1]
```diff
@@ -152,27 +152,35 @@ def validate_amount(self, field):
...
         self.project = project
+        self.edit = edit
 
     def validate_name(form, field):
         if field.data == form.name.default:
             raise ValidationError(_("User name incorrect"))
-        if Person.query.filter(Person.name == field.data)\
-                .filter(Person.project == form.project)\
-                .filter(Person.activated == True).all():
+        if not form.edit and \
```
Comment (almet) [inline budget/forms.py]: Rather than using `\`, can you use parenthesis?

Comment (almet) [inline budget/forms.py]: Rather than using `\`, can you use parenthesis?

**Comment by almet** [comment]
This looks good to me! (with small nits adressed)

**Comment by JocelynDelalande** [comment]
@almet ok, nitfixed :-)

File: budget/models.py [hunk 2]
```diff
@@ -217,9 +223,10 @@ def delete(self, project, id):
     archive = db.Column(db.Integer, db.ForeignKey("archive.id"))
 
     def pay_each(self):
-        """Compute what each person has to pay"""
+        """Compute what each share has to pay"""
 	if self.owers:
-		return self.amount / len(self.owers)
+                # FIXME: SQL might dot that more efficiently
```
Comment (almet) [inline budget/models.py]: Extra space here.

**Comment by almet** [comment]
Aaaand, it's merged!

**Pull request merged**